authorization/config.yaml

```diff
@@ -2,47 +2,16 @@ log:
   debug: false
 apis:
   grpc:
-    address: "0.0.0.0:9000"
+    address: "0.0.0.0:9100"
     tls:
       caPool: "/secrets/public/rootca.crt"
       keyFile: "/secrets/private/cert.key"
       certFile: "/secrets/public/cert.crt"
       clientCertificateRequired: true
-  http:
-    address: "0.0.0.0:9001"
-    tls:
-      caPool: "/secrets/public/rootca.crt"
-      keyFile: "/secrets/private/cert.key"
-      certFile: "/secrets/public/cert.crt"
-      clientCertificateRequired: true
-clients:
-  oauthClients:
-    device:
-      provider: "plgd"
-      clientID: ""
-      clientSecret: ""
-      scopes: []
-      authorizationURL: ""
-      tokenURL: ""
-      redirectURL: "" 
-      http:
-        maxIdleConns: 16
-        maxConnsPerHost: 32
-        maxIdleConnsPerHost: 16
-        idleConnTimeout: 30s
-        timeout: 10s
-        tls:
-          caPool: "/secrets/public/rootca.crt"
-          keyFile: "/secrets/private/cert.key"
-          certFile: "/secrets/public/cert.crt"
-          useSystemCAPool: false
-      ownerClaim: "sub"
-    client:
-      clientID: ""
-      scopes: []
-      authorizationURL: ""
-      audience: ""
-      redirectURL: ""
+    authorization:
+      authority:
+      audience:
+      ownerClaim: sub
```

Comment:
There will be also an HTTP endpoint to delete the device, or also list all user's devices. What's your argument then to have ownerClaim part of the grpc API?

It's the configuration of the service, how it organizes the data. It has nothing to do with the token validation which API does. It belongs to the core configuration of the service, not to API configuration as there might be 1..n of them.

It belongs to oauthClients.device

Reply:
So do you want to introduce the same property also for HTTP API of this service? Because when this will be authorized, there will be endpoints to list all users devices, delete user device, and others. Based on your logic described above, this property will need to be there as ownerID is acquired from the context - HTTP context in this case.

Reply:
HTTP API will be removed from authorization service.

Reply:
It won't because user will be at some point able to share his devices with different users or even more important, delete his own device what is a HTTP request from the UI to the authorization service.

Reply:
So do you want to introduce the same property also for HTTP API of this service?  - yes

Reply:
It is part of same authorization as is in GRPC.

Reply:
That means it doesnt belong to the specific API. If you have it both you might have a conflict. This value is used as a identifier when assigning devices to owners in database. That’s why it doesn’t belong to API but is a general service cfg.